package-lock.json

```diff
@@ -110,84 +110,39 @@
         "to-fast-properties": "^2.0.0"
       }
     },
-    "@types/body-parser": {
```

Comment:
Are all these changes in this file intentional?

Reply:
Not necessarily, I was trying to update package-lock.json to the new version so I did `npm install` is that not right?

Reply:
By doing that, you also updated the versions of all dependencies. Which is fine, but means you should do some more validation to ensure there aren't any regressions caused by that.

If you wanted to just update the version number, `npm version minor` or `npm version patch` would create a commit that just bumps the version. You could also edit both files manually